CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [ENHANCEMENT] Dashboards: Add config option `datasource_regex` to customise the regular expression used to select valid datasources for Mimir dashboards. #1802
 * [BUGFIX] Fix `container_memory_usage_bytes:sum` recording rule #1865
 * [BUGFIX] Fix `MimirGossipMembersMismatch` alerts if Mimir alertmanager is activated #1870
+* [BUGFIX] Fix regexp parsing panic for regexps with start/end cuantifiers. #1883
```

Comment:
```suggestion
* [BUGFIX] Fix regexp parsing panic for regexp label matchers with start/end quantifiers. #1883
```

Reply:
Done in https://github.com/grafana/mimir/pull/1883/commits/3f5a652f9a84ce86a283584ae197eac65c068a44